From: Ben Hutchings <ben@decadent.org.uk>
Subject: [PATCH] sched: Fix conflict between 2.6.32.7 and vserver

Make some cosmetic changes to sched.c on top of 2.6.32.7 so that
the vserver patch will still apply.

--- a/kernel/sched.c
+++ b/kernel/sched.c
@@ -1812,8 +1813,9 @@
 }
 #endif
 
+static void update_sysctl(void);
+
 static void calc_load_account_active(struct rq *this_rq);
-static void update_sysctl(void);
 
 #include "sched_stats.h"
 #include "sched_idletask.c"
